components/Balance.tsx

```diff
@@ -14,6 +15,11 @@ export default function Balance({
   balance = { formatted: "0", value: 0 },
 }: Props) {
   const { isConnected } = useAccount();
+  const [isMounted, setIsMounted] = useState(false);
+
+  useEffect(() => {
+    setIsMounted(true);
```

Comment:
what a world we live in that this is the lifecycle hack we need to use 😂 thank you react